hypha/static_src/src/app/src/containers/ReminderContainer/index.js

```diff
@@ -28,56 +28,44 @@ const styles = {
 
 class ReminderContainer extends React.PureComponent {
 
-    state = {
-        open : false
-    }
-
-    componentDidUpdate(prevProps) {
-        if(this.props.submissionID != prevProps.submissionID) {
-            this.props.initAction(this.props.submissionID)
-        }
-    }
+    renderSubmissionReminders = () => {
+        if(this.props.remindersLoading) return <LoadingPanel/> 
 
-    handleModalClose = () => {
-        this.setState({open : false})
+        if(this.props.reminders.length) {
+            return this.props.reminders.map(reminder =>
+                <ReminderList
+                    key={reminder.grouper}
+                    title={reminder.grouper}
+                    reminders={reminder.list} 
+                    deleteReminder={this.props.deleteReminder}
+                />)
+        }else return <div>No reminders yet.</div>
     }
 
     render(){
         const { classes } = this.props;
-        if(this.props.reminderInfo.loading) return <LoadingPanel /> 
         return (
             <div className="reminder-container">
                 <SidebarBlock title={"Reminders"}>
                     <div className="status-actions">
                         <button 
                             className="button button--primary button--half-width button--bottom-space reminder-button" 
-                            onClick={() => this.setState({open : true})}
+                            onClick={() => this.props.toggleModal(true)}
```

Comment:
Curious - any specific reason for moving to redux action instead of using component state?

Reply:
As we are using redux for state management, don't want to have local state. Just a code refactoring